bdns/dns_test.go

```diff
@@ -190,13 +188,31 @@ func serveLoopResolver(stopChan chan bool) chan bool {
 			fmt.Println(err)
 		}
 	}()
-	return waitChan
+}
+
+func pollServer() {
+	var conn *dns.Conn
+	timeout := time.Duration(time.Millisecond * 200)
+	tries := 25
+
+	for i := 0; conn == nil; i++ {
```

Comment:
Very cool, thanks @rolandshoemaker. I just saw this comment, I implemented the former comment, should I try this one?